en/docs/install-and-setup/upgrading-wso2-api-manager/260-to-410/upgrading-from-260-to-410.md

```diff
@@ -214,10 +214,6 @@ Follow the instructions below to move all the existing API Manager configuration
     
     Therefore, if registry versioning was enabled in WSO2 API-M 2.6.0 setup, it is **required** run the below scripts against **the database that is used by the registry**. Follow the below steps to achieve this.
     
-    !!! note "NOTE"
```

Comment:
@Lakith-Rambukkanage shall we remove this info box and make the verification step more visible. Otherwise it seems like an optional info, users may tend to skip it?

Reply:
Lets highlight the point that, registry versioning should be set to false from 4.0 onwards, even it was enabled in previous version.